.travis.yml

```
@@ -138,11 +138,16 @@ jobs:
       addons: sonarcloud
       script:
         - export STAGE=scan
-        # Various edits to the build.gradle file to get this to work. jhrg 1/22/24
-        # - travis_wait 30 gradle buildDists sonarqube -Dsonar.token=$SONAR_LOGIN -Dsonar.qualitygate.timeout=1800 -Dsonar.qualitygate.wait=true
-        - export SONAR_BRANCH_NAME="$TRAVIS_PULL_REQUEST_BRANCH"
-        - if test -z "$SONAR_BRANCH_NAME"; then export SONAR_BRANCH_NAME="$TRAVIS_BRANCH"; fi
-        - travis_wait 30 gradle sonar -Dsonar.login=$SONAR_LOGIN -Dsonar.qualitygate.timeout=1800 -Dsonar.qualitygate.wait=true -Dsonar.branch.name="$SONAR_BRANCH_NAME"
+        - | 
+          if test -z "$TRAVIS_PULL_REQUEST_BRANCH"; then 
```

```suggestion
          # multiline build instructions incoming!
          if test -z "$TRAVIS_PULL_REQUEST_BRANCH"; then 
```
as per @ndp-opendap's request :)